src/main/scala/mockws/FakeWSRequestHolder.scala

```diff
@@ -94,21 +101,21 @@ case class FakeWSRequestHolder(
       StreamedResponse(new FakeWSResponseHeaders(result), result.body.dataStream)
     }
 
-  @deprecated("2.5.0")
+  @deprecated("will be removed", "2.5.0")
   def streamWithEnumerator(): Future[(WSResponseHeaders, Enumerator[Array[Byte]])] =
     executeResult().map { r ⇒
       val headers = FakeWSResponseHeaders(r.header.status, r.header.headers.mapValues(e ⇒ Seq(e)))
       val source = r.body.dataStream.map(_.toArray)
       val publisher = source.runWith(Sink.asPublisher(false))
-      val enum: Enumerator[Array[Byte]] = Streams.publisherToEnumerator(publisher)
+      val enum: Enumerator[Array[Byte]] = IterateeStreams.publisherToEnumerator(publisher)
       headers → enum
     }
 
   private def executeResult(): Future[Result] = {
     logger.debug(s"calling $method $url")
 
     val action = routes.lift((method, url)).getOrElse(throw new Exception(s"no route defined for $method $url"))
-    def fakeRequest = FakeRequest(method, urlWithQueryParams()).withHeaders(headersSeq(): _*)
+    def fakeRequest = FakeRequest(method, urlWithQueryParams()).withHeaders(headersSeq(): _*).withBody(body)
```

Comment:
I don't really understand the change on the play side.
Does this mean that an `Action` does not produce the same result if used with a `FakeRequest` or with a "normal" request? It feels strange.
I have to dig further in play source code.